src/SurveyService.Validation/Option/EditOptionRequestValidator.cs

```diff
@@ -0,0 +1,69 @@
+using FluentValidation;
+using FluentValidation.Validators;
+using LT.DigitalOffice.Kernel.Validators;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Option;
+using LT.DigitalOffice.SurveyService.Validation.Option.Interfaces;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.SurveyService.Validation.Option;
+
+public class EditOptionRequestValidator : ExtendedEditRequestValidator<DbOption, EditOptionRequest>,  IEditOptionRequestValidator
+{
+  private void HandleInternalPropertyValidation(
+    Operation<EditOptionRequest> requestedOperation,
+    CustomContext context)
+  {
+    RequestedOperation = requestedOperation;
+    Context = context;
+    
+    #region paths
+    
+    AddСorrectPaths(
+      new List<string> 
+      {
+        nameof(EditOptionRequest.Content),
+        nameof(EditOptionRequest.IsActive)
+      });
+    
+    AddСorrectOperations(nameof(EditOptionRequest.Content), new() { OperationType.Replace });
+    AddСorrectOperations(nameof(EditOptionRequest.IsActive), new() { OperationType.Replace });
+    
+    #endregion
+
+    #region content
+
+    AddFailureForPropertyIf(
+      nameof(EditOptionRequest.Content),
+      x => x == OperationType.Replace,
+      new Dictionary<Func<Operation<EditOptionRequest>, bool>, string>
```

Comment:
it can be just 'new()'

Reply:
Well, in UserService full name is used 
![image](https://user-images.githubusercontent.com/50172336/201528296-e6a694b1-49df-4e4f-a6f8-edcc08ed623e.png)

Reply:
it's not wrong; it's just shorter with 'new()'
OK, let it be, if you want